lib/hocon/impl/config_null.rb

```diff
@@ -3,7 +3,13 @@
 require 'hocon/impl'
 require 'hocon/config_value_type'
 
-class Hocon::Impl::ConfigNull < Hocon::Impl::AbstractConfigValue
+class Hocon::Impl::ConfigNull
+  include Hocon::Impl::AbstractConfigValue
+
+  def initialize(origin)
+    super(origin)
```

Comment:
Actually, it must work, since we're parsing configurations will `null` in them in tests and they aren't failing.